x-pack/packages/kbn-elastic-assistant-common/impl/capabilities/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Interface for features available to the elastic assistant
+ */
+export type AssistantFeatures = { [K in keyof typeof assistantFeatures]: boolean };
+
+/**
+ * Default features available to the elastic assistant
+ */
+export const assistantFeatures = Object.freeze({
```

Comment:
nit: If `assistantFeatures` is always used when the canonical default values of the `AssistantFeatures` type are required, e.g. the following usages:

```ts
const features = this.registeredFeatures?.get(pluginName) ?? assistantFeatures;

const { assistantModelEvaluation: modelEvaluatorEnabled, assistantStreamingEnabled } = capabilities ?? assistantFeatures;

const { assistantModelEvaluation: modelEvaluatorEnabled, assistantStreamingEnabled } =
    capabilities ?? assistantFeatures;
```



 consider a name like `defaultAssistantFeatures` if, for example, the current usage:

```ts
const features = this.registeredFeatures?.get(pluginName) ?? assistantFeatures;
```

would be improved by:

```ts
const features = this.registeredFeatures?.get(pluginName) ?? defaultAssistantFeatures;
```

Reply:
Resolved in [125109d](https://github.com/elastic/kibana/pull/174317/commits/125109daf4dc8c7556729bbb10055b6add21028d)